packages/rulesets/src/spectral/functions/validate-patch-body-param-properties.ts

```diff
@@ -46,19 +46,39 @@ export const validatePatchBodyParamProperties = createRulesetFunction<unknown, O
         _opts.should.forEach((p: string) => {
           if (!getProperties(bodyParameter)?.[p] && getProperties(responseSchema)?.[p]) {
             errors.push({
-              message: `The patch operation body parameter schema should contains property '${p}'.`,
+              message: `The patch operation body parameter schema should contain property '${p}'.`,
               path: [...path, "parameters", index],
             })
           }
         })
       }
       if (_opts.shouldNot) {
         _opts.shouldNot.forEach((p: string) => {
-          if (getProperties(bodyParameter)?.[p]) {
-            errors.push({
-              message: `The patch operation body parameter schema should not contains property ${p}.`,
-              path: [...path, "parameters", index],
-            })
+          const property = getProperties(bodyParameter)?.[p];
+          if (property) {
+            let isPropertyBoolean = false;
+            let isPropertyImmutable = false;
+
+            if (property["readOnly"] && property["readOnly"] === true)
+            {
+              isPropertyBoolean = true;
```

Comment:
Inconsistent indentation: sometimes I see indentation of 4, sometimes of 2.

Similar problem with bracketing style. Sometimes I see opening `{` on the same line, sometimes on the next one.

Let's be consistent. I think this codebase has built-in formatter, let's use it. Here it is:
https://github.com/Azure/azure-openapi-validator/blob/aa7c987f73d163cb5f2810a5323a61bb869bb4e5/.vscode/settings.json#L1C1-L8